### PR TITLE
[BM] Fix multus deployment 

### PR DESCRIPTION
**Pull request by OdedViner**

### CODE REVIEW

File: ocs_ci/deployment/deployment.py [hunk 1]
```diff
@@ -803,10 +803,12 @@ def deploy_ocs_via_operator(self, image=None):
                 interfaces.add(config.ENV_DATA["multus_cluster_net_interface"])
             worker_nodes = get_worker_nodes()
             node_obj = ocp.OCP(kind="node")
-            for node in worker_nodes:
-                for interface in interfaces:
-                    ip_link_cmd = f"ip link set promisc on {interface}"
-                    node_obj.exec_oc_debug_cmd(node=node, cmd_list=[ip_link_cmd])
+            platform = config.ENV_DATA.get("platform").lower()
+            if platform == "baremetal":
```
Comment (clacroix12) [inline ocs_ci/deployment/deployment.py:807]: We want to skip if baremetal right?

```suggestion
            if platform != "baremetal":
```

**Review by petr-balogh — APPROVED** [review]

**Review by clacroix12 — APPROVED** [review]
LGTM

**Comment by OdedViner** [comment]
Deployment job pass https://url.corp.redhat.com/3466255
@petr-balogh @clacroix12 Can we merge it?

**Comment by vavuthu** [comment]
@OdedViner slow ops reported. please check https://url.corp.redhat.com/256bd1c

File: ocs_ci/deployment/deployment.py [hunk 2]
```diff
@@ -803,10 +803,12 @@ def deploy_ocs_via_operator(self, image=None):
                 interfaces.add(config.ENV_DATA["multus_cluster_net_interface"])
             worker_nodes = get_worker_nodes()
             node_obj = ocp.OCP(kind="node")
-            for node in worker_nodes:
-                for interface in interfaces:
-                    ip_link_cmd = f"ip link set promisc on {interface}"
-                    node_obj.exec_oc_debug_cmd(node=node, cmd_list=[ip_link_cmd])
+            platform = config.ENV_DATA.get("platform").lower()
+            if platform != "baremetal":
```
[the remark this answers is not in the document]
Comment (OdedViner) [reply ocs_ci/deployment/deployment.py:807]: done

File: conf/deployment/baremetal/upi_1az_rhcos_multus_public_nvme_intel_3m_3w.yaml [hunk 3]
```diff
@@ -12,6 +12,7 @@ ENV_DATA:
   disk_pattern: 'nvme-INTEL'
   is_multus_enabled: true
   multus_public_net_interface: 'enp1s0f0'
+  multus_cluster_net_interface: 'enp1s0f0'
```
[the remark this answers is not in the document]
Comment (OdedViner) [reply conf/deployment/baremetal/upi_1az_rhcos_multus_public_nvme_intel_3m_3w.yaml:15]: yes.. it is the same physical interface...

**Comment by OdedViner** [comment]
> @OdedViner slow ops reported. please check https://url.corp.redhat.com/256bd1c

@vavuthu I don't know the reason for this failure  
CC @petr-balogh @ebenahar

**Review by prsurve — APPROVED** [review]

**Review by petr-balogh — APPROVED** [review]

**Review by clacroix12 — DISMISSED** [review]

File: conf/deployment/baremetal/upi_1az_rhcos_multus_cluster_nvme_intel_3m_3w.yaml [hunk 4]
```diff
@@ -12,6 +12,7 @@ ENV_DATA:
   disk_pattern: 'nvme-INTEL'
   is_multus_enabled: true
   multus_public_net_interface: 'enp1s0f0'
+  multus_cluster_net_interface: 'enp1s0f0'
```
Comment (ebenahar) [inline conf/deployment/baremetal/upi_1az_rhcos_multus_cluster_nvme_intel_3m_3w.yaml:15]: shouldn't we change it to enp1s0f1?

Comment (OdedViner) [reply conf/deployment/baremetal/upi_1az_rhcos_multus_cluster_nvme_intel_3m_3w.yaml:15]: yes.. if it is woking...I will test it again manually..

**Review by ebenahar — APPROVED** [review]

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**